lib/active_merchant/billing/gateways/netbanx.rb

```diff
@@ -240,6 +245,48 @@ def headers
+            when '3024' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined because the issuing bank does not permit the transaction for this card.
+            when '3032' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined by the issuing bank or external gateway because the card is probably in one of their negative databases.
+            when '3035' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined due to exceeded PIN attempts.
+            when '3036' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined due to an invalid issuer.
+            when '3037' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined because it is invalid.
+            when '3038' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined due to customer cancellation.
+            when '3039' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined due to an invalid authentication value.
+            when '3040' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined because the request type is not permitted on the card.
+            when '3041' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined due to a timeout.
+            when '3042' ; STANDARD_ERROR_CODE[:card_declined] # Your request has been declined due to a cryptographic error.
+            when '3045' ; STANDARD_ERROR_CODE[:invalid_expiry_date] # You submitted an invalid date format for this request.
+            when '3046' ; STANDARD_ERROR_CODE[:card_declined] # The transaction was declined because the amount was set to zero.
+            when '3047' ; STANDARD_ERROR_CODE[:card_declined] # The transaction was declined because the amount exceeds the floor limit.
+            when '3048' ; STANDARD_ERROR_CODE[:card_declined] # The transaction was declined because the amount is less than the floor limit.
+            when '3049' ; STANDARD_ERROR_CODE[:card_declined] # The bank has requested that you retrieve the card from the cardholder – the credit card has expired.
+            when '3050' ; STANDARD_ERROR_CODE[:card_declined] # The bank has requested that you retrieve the card from the cardholder – fraudulent activity is suspected.
+            when '3051' ; STANDARD_ERROR_CODE[:card_declined] # The bank has requested that you retrieve the card from the cardholder – contact the acquirer for more information.
+            when '3052' ; STANDARD_ERROR_CODE[:card_declined] # The bank has requested that you retrieve the card from the cardholder – the credit card is restricted.
+            when '3053' ; STANDARD_ERROR_CODE[:card_declined] # The bank has requested that you retrieve the card from the cardholder – please call the acquirer.
+            when '3054' ; STANDARD_ERROR_CODE[:card_declined] # The transaction was declined due to suspected fraud.
```

Comment:
Style nitpick, so not important, but I'd typically prefer `when ... then` syntax rather than a semicolon. We don't have any enforced linter rules (yet), so it's all right for now.